### ISSUE TITLE
Email validation causes error on submit

### ISSUE THREAD

**Issue by dowadidi** [verbb/formie#832]
**Description**
When submitting a form containing an email field, you get the following error: `Setting unknown property: yii\validators\EmailValidator::enableLocalIDN`.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: Yes

**Additional info**
- Plugin version: **1.5.14**
- Craft version: **3.6.18**
- Multi-site: Yes

**Additional context**
Not sure, but might have something to do with: https://github.com/verbb/formie/commit/ccf95107bae9d7d5a3ea0bed23231307f1c1a253

**Stacktrace**
```yii\base\UnknownPropertyException: Setting unknown property: yii\validators\EmailValidator::enableLocalIDN in /var/www/vendor/yiisoft/yii2/base/Component.php:210
Stack trace:
#0 /var/www/vendor/yiisoft/yii2/BaseYii.php(558): yii\base\Component->__set('enableLocalIDN', false)
#1 /var/www/vendor/yiisoft/yii2/base/BaseObject.php(107): yii\BaseYii::configure(Object(yii\validators\EmailValidator), Array)
#2 [internal function]: yii\base\BaseObject->__construct(Array)
#3 /var/www/vendor/yiisoft/yii2/di/Container.php(420): ReflectionClass->newInstanceArgs(Array)
#4 /var/www/vendor/yiisoft/yii2/di/Container.php(171): yii\di\Container->build('yii\\validators\\...', Array, Array)
#5 /var/www/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('yii\\validators\\...', Array, Array)
#6 /var/www/vendor/yiisoft/yii2/validators/Validator.php(231): yii\BaseYii::createObject(Array)
#7 /var/www/vendor/yiisoft/yii2/base/Model.php(469): yii\validators\Validator::createValidator('yii\\validators\\...', Object(verbb\formie\elements\Submission), Array, Array)
#8 /var/www/vendor/yiisoft/yii2/base/Model.php(421): yii\base\Model->createValidators()
#9 /var/www/vendor/yiisoft/yii2/base/Model.php(190): yii\base\Model->getValidators()
#10 /var/www/vendor/yiisoft/yii2/base/Model.php(355): yii\base\Model->scenarios()
#11 /var/www/vendor/verbb/formie/src/elements/Submission.php(446): yii\base\Model->validate(NULL, true)
#12 /var/www/vendor/verbb/formie/src/controllers/SubmissionsController.php(488): verbb\formie\elements\Submission->validate()
#13 [internal function]: verbb\formie\controllers\SubmissionsController->actionSubmit()
#14 /var/www/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#15 /var/www/vendor/yiisoft/yii2/base/Controller.php(181): yii\base\InlineAction->runWithParams(Array)
#16 /var/www/vendor/craftcms/cms/src/web/Controller.php(190): yii\base\Controller->runAction('submit', Array)
#17 /var/www/vendor/yiisoft/yii2/base/Module.php(534): craft\web\Controller->runAction('submit', Array)
#18 /var/www/vendor/craftcms/cms/src/web/Application.php(278): yii\base\Module->runAction('formie/submissi...', Array)
#19 /var/www/vendor/craftcms/cms/src/web/Application.php(591): craft\web\Application->runAction('formie/submissi...', Array)
#20 /var/www/vendor/craftcms/cms/src/web/Application.php(257): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#21 /var/www/vendor/yiisoft/yii2/base/Application.php(392): craft\web\Application->handleRequest(Object(craft\web\Request))
#22 /var/www/web/index.php(28): yii\base\Application->run()
#23 {main}```

**Comment by engram-design** [comment]
Sorry about that, looks like I didn't realise that option was only added in Craft 3.7.9/Yii 2.0.23. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.14",
  "...": "..."
}
```

Then run `composer update`.

**Comment by dowadidi** [comment]
@engram-design That did the trick. Thanks!

**Comment by engram-design** [comment]
Fixed in [1.5.15](https://github.com/verbb/formie/releases/tag/1.5.15)
